src/Components/Web/src/Virtualization/Virtualize.cs

```diff
@@ -193,6 +199,11 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
 
         builder.OpenRegion(3);
 
+        if (EmptyContent is not null && _itemCount == 0)
```

Comment:
Wouldn't this show the empty content when the list is still being loaded in the event the data source is an `ItemsProvider`? That doesn't seem right.

Could we also add some tests for this?

Reply:
You're right, it indeed does. I will rectify that and write some tests.